test/test_inverse.jl

```diff
@@ -90,6 +90,9 @@ InverseFunctions.inverse(f::Bar) = Bar(inv(f.A))
     InverseFunctions.test_inverse(inverse(Base.Fix2(divrem, 5)), (-3, -2); compare=(==))
     InverseFunctions.test_inverse(inverse(Base.Fix2(fldmod, 5)), (-3, 2); compare=(==))
 
+    InverseFunctions.test_inverse(reim, -3; compare=(==))
```

Comment:
Shouldn't we also test the inverse function (ie. `splat(complex)`)?

Reply:
> Shouldn't we also test the inverse function (ie. splat(complex))

Good point, yes - could you add a test for that, @aplavin ?